src/test/java/reposense/parser/RepoConfigurationTest.java

```diff
@@ -21,49 +22,53 @@
+    private static final List<String> THIRD_AUTHOR_ALIASES = Arrays.asList("Jordan Chong");
+    private static final List<String> FOURTH_AUTHOR_ALIASES = Arrays.asList("Tianwei");
 
+    private static final List<String> REPO_LEVEL_GLOB_LIST = Arrays.asList("*.pyc");
+    private static final List<String> FIRST_AUTHOR_GLOB_LIST = Arrays.asList("*.aa1", "**.aa2");
+    private static final List<String> SECOND_AUTHOR_GLOB_LIST = Collections.emptyList();
 
     @Test
-    public void configJson_overridesRepoConfig_success()
-            throws InvalidLocationException, FileNotFoundException, GitDownloaderException {
-        RepoConfiguration expectedConfig = new RepoConfiguration(TEST_REPO_DELTA, "master");
-
-        List<Author> authors = new ArrayList<Author>();
-        authors.add(FIRST_AUTHOR);
-        authors.add(SECOND_AUTHOR);
-        authors.add(THIRD_AUTHOR);
-        authors.add(FOURTH_AUTHOR);
-
-        expectedConfig.setAuthorList(authors);
-
-        expectedConfig.setAuthorAliases(FIRST_AUTHOR, FIRST_AUTHOR_ALIASES);
-        expectedConfig.setAuthorAliases(SECOND_AUTHOR, SECOND_AUTHOR_ALIASES);
-        expectedConfig.setAuthorAliases(THIRD_AUTHOR, THIRD_AUTHOR_ALIASES);
-        expectedConfig.setAuthorAliases(FOURTH_AUTHOR, FOURTH_AUTHOR_ALIASES);
+    public void configJson_overridesRepoConfig_success() throws InvalidLocationException, GitDownloaderException {
+        FIRST_AUTHOR.setIgnoreGlobList(FIRST_AUTHOR_GLOB_LIST);
+        SECOND_AUTHOR.setIgnoreGlobList(SECOND_AUTHOR_GLOB_LIST);
+        THIRD_AUTHOR.setIgnoreGlobList(REPO_LEVEL_GLOB_LIST);
+        FOURTH_AUTHOR.setIgnoreGlobList(REPO_LEVEL_GLOB_LIST);
+
+        List<Author> expectedAuthors = new ArrayList<Author>();
+        expectedAuthors.add(FIRST_AUTHOR);
+        expectedAuthors.add(SECOND_AUTHOR);
+        expectedAuthors.add(THIRD_AUTHOR);
+        expectedAuthors.add(FOURTH_AUTHOR);
 
-        expectedConfig.setAuthorDisplayName(FIRST_AUTHOR, "Ahm");
-        expectedConfig.setAuthorDisplayName(SECOND_AUTHOR, "Cod");
-        expectedConfig.setAuthorDisplayName(THIRD_AUTHOR, "Jor");
-        expectedConfig.setAuthorDisplayName(FOURTH_AUTHOR, "Loh");
-
-        expectedConfig.setAuthorAliases(FIRST_AUTHOR, "Ahmad Syafiq");
-        expectedConfig.setAuthorAliases(SECOND_AUTHOR, "Codee");
-        expectedConfig.setAuthorAliases(THIRD_AUTHOR, "Jordan Chong");
-        expectedConfig.setAuthorAliases(FOURTH_AUTHOR, "Tianwei");
+        RepoConfiguration expectedConfig = new RepoConfiguration(TEST_REPO_DELTA, "master");
+        expectedConfig.setAuthorList(expectedAuthors);
+        expectedConfig.addAuthorAliases(FIRST_AUTHOR, FIRST_AUTHOR_ALIASES);
+        expectedConfig.addAuthorAliases(SECOND_AUTHOR, SECOND_AUTHOR_ALIASES);
+        expectedConfig.addAuthorAliases(THIRD_AUTHOR, THIRD_AUTHOR_ALIASES);
+        expectedConfig.addAuthorAliases(FOURTH_AUTHOR, FOURTH_AUTHOR_ALIASES);
+        expectedConfig.addAuthorDisplayName(FIRST_AUTHOR, "Ahm");
```

Comment:
Have a line break between this 2 setting up method calls, just to make it cleaner